spring-cloud-task-samples/batch-events/pom.xml

```diff
@@ -50,11 +51,13 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-stream-rabbit</artifactId>
+			<version>1.0.2.RELEASE</version>
```

Comment:
Something to talk about but definitely not something we want to do for this PR...